lib/kitchen/directions/bake_notes/bake_autotitled_notes.rb

```diff
@@ -8,6 +8,14 @@ def self.v1(book:, classes:)
           next unless (note.classes & classes).any?
 
           bake_note(note: note)
+
+          next unless note.notes.present?
```

Comment:
is this needed?  if there are no `note.notes` then the following loop will do nothing anyway